packages/client/src/App.tsx

```diff
@@ -1,14 +1,35 @@
-import { ApolloClient, ApolloProvider, InMemoryCache } from '@apollo/client'
+import { ApolloClient, ApolloProvider, gql, InMemoryCache, useQuery } from '@apollo/client'
 import AccountView from '@components/AccountView'
 import MapSlideUpSheet from '@components/MapSlideUpSheet'
 import MapView from '@components/MapView'
+import DistanceRadiusSelector, { DISTANCES, INITIAL_DISTANCE_INDEX } from '@components/SortSearchResults/DistanceRadiusSelector'
+import SearchResultSorter from '@components/SortSearchResults/SearchResultSorter'
 import { Business, BusinessType, DisplayableBusiness, Location } from '@futureproof/typings'
 import { NavigationContainer } from '@react-navigation/native'
 import { createStackNavigator, StackScreenProps } from '@react-navigation/stack'
 import Constants from 'expo-constants'
-import React from 'react'
-import { SafeAreaView, StyleSheet, TouchableOpacity, View } from 'react-native'
-import AccountButton from './components/AccountButton'
+import React, { useState } from 'react'
+import { Dimensions, SafeAreaView, StyleSheet, TouchableOpacity, View } from 'react-native'
+import AccountButton from '@components/AccountButton'
+import { Option } from '@components/SortSearchResults/OptionList'
+
+const GET_COORDINATES = gql `
+  query {
+    locations {
+      id
+      latitude
+      longitude
+      business {
+        sustainabilityScore
+      }
+    }
+  }`
+
+export type LocationType = Pick<Location, 'latitude' | 'longitude' | 'id'>;
+
+export interface LocationTypeWithRating extends LocationType {
+  business : Pick<Business, 'sustainabilityScore'>
+}
 
 const ExampleBusiness : DisplayableBusiness =  {
```

Comment:
Do we need this anymore?

Reply:
not sure i just moved it when refactoring